### DISCUSSION TITLE
Ubuntu Python - OW doesn´t show Scan docs option

### DISCUSSION THREAD

**Discussion by raullopezgn** [open-webui/open-webui#7118] [General]
# Bug Report

I need that Open Webui (last version installed) scans the docs of a folder to "feed" LLMs knowledge, however when I try follow the following path for do it, there is no "scan" option:
**Admin Panel > Settings > Documents > Scan**

At the "Documents" section I have:
Embedding Model Engine: Ollama
Embedding Model: paraphrase-multilingual:latest

One last question, where is located the folder, where OW would scan documents?

---
## Installation Method
I created an environment with Conda for Open Webui (OW) with Python 3.11. I installed and executed OW using the commands:
pip install open-webui
open-webui serve

## Environment
- **Open WebUI Version:** v0.4.1
- **Ollama:** v0.3.14

- **Operating System:** Ubuntu 22.04.5
- **Browser:** Firefox 132.0.2

**Confirmation:**
- [X ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:
I would need to see the "Scan" button in the following path:
**Admin Panel > Settings > Documents > Scan**

## Actual Behavior:
There isn´t any button or option for scanning docs to feed LLM.

**Comment by mahenning** [comment]
I got removed
https://github.com/open-webui/open-webui/discussions/6013#discussioncomment-10895978
